docs/api-guide/fields.md

```diff
@@ -264,6 +264,7 @@ Corresponds to `django.db.models.fields.DecimalField`.
 - `coerce_to_string` Set to `True` if string values should be returned for the representation, or `False` if `Decimal` objects should be returned. Defaults to the same value as the `COERCE_DECIMAL_TO_STRING` settings key, which will be `True` unless overridden. If `Decimal` objects are returned by the serializer, then the final output format will be determined by the renderer.
 - `max_value` Validate that the number provided is no greater than this value.
 - `min_value` Validate that the number provided is no less than this value.
+- `localize` Set to `True` to enable localization of input and output based on the current locale. Defaults to `False`. Note that data formatting is enabled if you have set `USE_L10N=True` in your settings file.
```

Comment:
Presumable we should also ensure that setting `localize` *forces* coerce_to_string to be `True`, right?